### PR TITLE
Var renames to be consistent with the `tf_` and `aws_` groups

### PR DESCRIPTION
**Pull request by arm4b**
* Renames:
  * `ec2_instance_type` -> `aws_ec2_instance_type`
  * `ec2_instance_profile` -> `aws_ec2_instance_profile`
  * `stack_destroy` -> `tf_stack_destroy`
* Reorder and update the GHA inputs/descriptions to be in sync with the `README`

### CODE REVIEW

**Comment by PhillypHenning** [comment]
Did you ensure that these aren't being populated in the tf.vars and used by Terraform?

**Comment by mcascone** [comment]
I appreciate this because i am working on the GCP implementation of this stuff, and it's important to keep the distinctions separate.

File: action.yaml [hunk 1]
```diff
@@ -89,15 +87,10 @@ runs:
         TF_STATE_BUCKET_DESTROY: ${{ inputs.tf_state_bucket_destroy }}
 
         APP_PORT: ${{ inputs.app_port }}
-        EC2_INSTANCE_PROFILE: ${{ inputs.ec2_instance_profile }}
-        EC2_INSTANCE_TYPE: ${{ inputs.ec2_instance_type }}
-        STACK_DESTROY: ${{ inputs.stack_destroy }}
+        EC2_INSTANCE_PROFILE: ${{ inputs.aws_ec2_instance_profile }}
+        EC2_INSTANCE_TYPE: ${{ inputs.aws_ec2_instance_type }}
+        STACK_DESTROY: ${{ inputs.tf_stack_destroy }}
```
[the remark this answers is not in the document]
Comment (PhillypHenning) [reply action.yaml:92]: Understood. 
I suppose my question back to you in that regard is; 

Would you see your opinion change if we introduced a new cloud provider? 
If we added GCP support, would you want to prefix the tf vars with "aws_`

[the remark this answers is not in the document]
Comment (LeoDiazL) [reply action.yaml:92]: Although this is created by terraform, it's referring to the whole stack, not only the AWS part.

**Review by PhillypHenning — APPROVED** [review]

**Pull request merged**